test/support/gencert.sh

```diff
@@ -1,5 +1,12 @@
 #! /bin/sh
 set -ex
 
-openssl req -x509 -nodes -newkey rsa:2048 -keyout ./test/fixtures/server.key -out ./test/fixtures/server.crt -days 3650 \
--subj "/C=US/ST=Illinois/L=Chicago/O=node-express-session/CN=express-session.local"
+OS=$(uname)
+
+if [ "$OS" = "Linux" ] || [ "$OS" = "Darwin" ]; then
```

Comment:
Would it be better to reverse this if? I mean, should we check that if the OS is Windows, then use the alternative command.

The CI currently tests in windows and linux, but BSD and other similar systems are supported as far as I know.

Reply:
Easy switch case could be:

```sh
case "$(uname)" in
  MINGW*) ... ;;
  *) ... ;;
esac
```